build/yamls/base/crds.yml

```diff
@@ -34,3 +34,125 @@ spec:
     kind: AntreaAgentInfo
     shortNames:
       - aai
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusternetworkpolicies.crd.antrea.tanzu.vmware.com
+spec:
+  group: crd.antrea.tanzu.vmware.com
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+  scope: Cluster
+  names:
+    plural: clusternetworkpolicies
+    singular: clusternetworkpolicy
+    kind: ClusterNetworkPolicy
+    shortNames:
+      - cnp
+  # Prune any unknown fields
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      properties:
+        spec:
+          # Ensure that Spec.AppliedTo and Spec.Priority fields are set
+          required:
+            - appliedTo
+            - priority
+          type: object
+          properties:
+           priority:
+             type: integer
+             # Ensure that Spec.Priority field is between 1 and 10
+             minimum: 1
+             maximum: 10
+           appliedTo:
```

Comment:
this block should ensure that no other selector except podSelector and/or namespaceSelector is allowed as part of AppliedTo. @jianjuns

Reply:
Got it.